fmriprep/workflows/bold/resampling.py

```diff
@@ -462,6 +464,7 @@ def init_bold_std_trans_wf(
     output_names = [
         "bold_mask_std",
         "bold_std",
+        "t2star_std",
```

Comment:
i think the problem is this is always being joined on, rather than only conditionally being in this list when multiecho is True

Reply:
Ah, I was wondering why we made the FreeSurfer ones conditional rather than just letting them be `Undefined`.